packages/manager/modules/pci/src/projects/project/private-registry/create/create.controller.js

```diff
@@ -0,0 +1,42 @@
+import get from 'lodash/get';
+import { REGION } from '../private-registry.constants';
+
+export default class {
+  /* @ngInject */
+  constructor($state, $translate, pciPrivateRegistryService) {
+    this.$state = $state;
+    this.$translate = $translate;
+    this.privateRegistryService = pciPrivateRegistryService;
+    this.isLoading = false;
+    this.REGION = REGION;
+    this.registry = {};
+    this.showAcceptOption = false;
+  }
+
+  $onInit() {
+    if (!this.$state.current.name.includes('onboarding')) {
+      this.showAcceptOption = true;
+    }
+  }
+
+  createRegistry() {
+    this.isLoading = true;
+    this.registry.region = this.REGION; // only GRA7 supported as of now
```

Comment:
You should remove this comment, it's not on our side and if GRA1 is supported one day, we are going to forget removing this comment ^.^